textdb/textdb-web/src/main/java/edu/uci/ics/textdb/web/request/beans/OperatorLinkBean.java

```diff
@@ -35,4 +38,24 @@ public String getToOperatorID() {
     public void setToOperatorID(String toOperatorID) {
         this.toOperatorID = toOperatorID;
     }
+
+    @Override
+    public boolean equals(Object other) {
```

Comment:
I did not see a need for the `equals()` function on the `OperatorLinkBean` class.